src/platform/testing/e2e/nightwatch-commands/evaluateRadioButtonsReverse.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+
+/**
+ * Evaluate radio buttons for basic reverse keyboard functionality:
+ *
+ * Radio groups can receive keyboard focus by pressing TAB
+ * Each radio button can be checked from bottom to top by pressing
+ * the UP or LEFT arrow keys.
+ *
+ * ```javascript
+ *  this.demoTest = function (client) {
+ *    client.evaluateRadioButtonsReverse([
+ *      "input[type="radio"]#1",
+ *      "input[type="radio"]#2",
+ *      "input[type="radio"]#3"
+ *    ], client.Keys.ARROW_UP);
+ *  };
+ * ```
+ *
+ * @method evaluateRadioButtonsReverse
+ * @param {string} [selectorArray] The array of radio buttons to be evaluated
+ * @param {object} [arrowPressed] Nightwatch Keys object. Expects ARROW_UP || ARROW_LEFT.
+ * @api commands
+ */
+exports.command = function evaluateRadioButtonsReverse(
+  selectorArray,
+  arrowPressed,
+) {
+  const arrLength = selectorArray.length;
+  const element = selectorArray[0];
+  const client = this;
+
+  return client.waitForElementPresent(element, 1000, function() {
+    for (let i = arrLength - 1; i >= 0; i -= 1) {
+      this.keys(arrowPressed).assert.isActiveElement(selectorArray[i]);
+    }
```

Comment:
Alternatively:
```js
selectorArray
  .reverse()
  .forEach(s => this.keys(arrowPressed).assert.isActiveElement(s));
```

Reply:
Ideally, I think you wouldn't care about the order, but I don't understand what this is doing, so I assume you have to care. :man_shrugging:

Reply:
Radio buttons are sequential when you're interacting with them by keypress. These methods are gathering the array, asserting each index is focused, pressing the arrow key, then re-asserting focus. I hadn't seen the `reverse()` method before, so having that is nice and allows for more functional means of achieving the same goal.